hilbert/factories.py

```diff
@@ -88,18 +141,29 @@ def build_mle_sample_solver(
         temperature=2,            # MLE option
         batch_size=10000,
         balanced=False,
+        gibbs=False,
         bias=False,
         init_embeddings_path=None,
         dimensions=300,
         learning_rate=0.01,
         opt_str='adam',
+        scheduler_str=None,
+        constant_fraction=1,
```

Comment:
Just need some indication that this is related to the learning rate scheduler.
```suggestion
        lr_scheduler_constant_fraction=1,
```